openshift/settings.bc.sh

```diff
@@ -1,4 +1,5 @@
 # Setting for the TheOrgBook-BC environments.
-# Uses the existing `devex-von-tools` environment for builds and images
-export TOOLS="devex-von-tools"
-export PROJECT_NAMESPACE="devex-von-bc-tob"
\ No newline at end of file
+# Uses the existing `8ad0ea-tools` environment for builds and images
+export TOOLS="8ad0ea-tools"
+export PROJECT_NAMESPACE="8ad0ea"
+export SKIP_PIPELINE_PROCESSING=1
```

Comment:
Why are we skipping pipeline processing for the bc profile?

Reply:
Because I thought pipelines were on hold until we decided whether we were using them as-is, or migrating to other CI/CD predicaments. I am skipping the processing here because otherwise it would have caused a lot of noise, but I can revert this change if it is unwanted.